src/Makefile.bench.include

```diff
@@ -72,7 +71,7 @@ bench_bench_bitcoin_SOURCES += bench/wallet_balance.cpp
 endif
 
 bench_bench_bitcoin_LDADD += $(BOOST_LIBS) $(BDB_LIBS) $(EVENT_PTHREADS_LIBS) $(EVENT_LIBS) $(MINIUPNPC_LIBS)
-bench_bench_bitcoin_LDFLAGS = $(RELDFLAGS) $(AM_LDFLAGS) $(LIBTOOL_APP_LDFLAGS)
+bench_bench_bitcoin_LDFLAGS = $(RELDFLAGS) $(AM_LDFLAGS) $(LIBTOOL_APP_LDFLAGS) -levent
```

Comment:
Please use `$(EVENT_LIBS)` instead of `-levent` directly.

Reply:
Code updated.